[Testing] lit-html missing

Just followed this tutorial in an empty folder : https://open-wc.org/testing/

Got this error :

`
ERROR in ./node_modules/@open-wc/testing-helpers/fixture-no-side-effect.js
Module not found: Error: Can't resolve 'lit-html' in '/xxx/node_modules/@open-wc/testing-helpers'
 @ ./node_modules/@open-wc/testing-helpers/fixture-no-side-effect.js 1:0-42 16:26-40 33:26-40
 @ ./node_modules/@open-wc/testing-helpers/fixture.js
 @ ./node_modules/@open-wc/testing-helpers/index.js
 @ ./node_modules/@open-wc/testing/index.js
 @ ./test/test-open-wc-stencil.test.js
`

Run `npm i --save-dev lit-html` solve the error.

--- a/packages/testing-helpers/README.md
+++ b/packages/testing-helpers/README.md
@@ -8,6 +8,14 @@ In order to efficiently test Web Components you will need some helpers to regist
 This is part of the default [open-wc](https://open-wc.org/) recommendation
 :::
 
+::: warning
+Testing helpers requires as a peer dependency [lit-html](https://lit-html.polymer-project.org/).
+You can install it inside your project with npm :
+```bash
+npm i --save lit-html
+```
+:::
+
 ## Test a custom element
 ```js
 import { fixture } from '@open-wc/testing-helpers';

